integration/mediation-tests/tests-sample/src/test/resources/testng.xml

```diff
@@ -68,6 +68,11 @@
                     <exclude name=".*"/>
                 </methods>
             </class>
+            <class name="org.wso2.carbon.esb.samples.test.miscellaneous.Sample650TestCase">
+                <methods>
+                    <exclude name="testLocalEntries"/>
```

Comment:
Shall we create a GitHub issue to track the exclusion of this? Otherwise, we can miss adding this back later.

Reply:
Have we created an issue to track this?

Reply:
we have created an issue. Please refer: https://github.com/wso2/micro-integrator/issues/366